mlflow/tracking/fluent.py

```diff
@@ -702,6 +715,48 @@ def log_params(params: Dict[str, Any]) -> None:
     MlflowClient().log_batch(run_id=run_id, metrics=[], params=params_arr, tags=[])
 
 
+def log_input(dataset: Dataset, context: str) -> None:
```

Comment:
Can we also accept a dictionary of additional tags? 

```suggestion
def log_input(dataset: Dataset, context: str = None, tags: Dict[str, str]) -> None:
```

Reply:
(Context is also optional)